### ISSUE TITLE
Accessibility: keyboard navigation in properties panel is difficult with spinboxes

### ISSUE THREAD

**Issue by MarcSabatella** [musescore/MuseScore#16739]
### Issue type

UI bug

### Bug description

When using the keyboard to navigate in Properties, the cursor keys are the usual controls to navigate with a group, but this doesn't work on spinboxes, because the cursor instead navigates the text within (left/right) or changes the value (up/down).   Tab does not work either.

### Steps to reproduce

1. select a note in a score
2. open Properties panel
3. open the Appearance tab
4. click in the Leading spinbox
5. attempt to navigate by keyboard to the next spinbox (Measure width)

Result: neither Right, Down, nor Tab get you there.  Only Esc or Enter followed by Right do.

### MuseScore Version

OS: Debian GNU/Linux 11 (bullseye), Arch.: x86_64, MuseScore version (64-bit): 4.0.2-230690504, revision: github-musescore-musescore-dbe7c6d

### Regression

Yes, this used to work in Musescore 3.x and now is broken

### Operating system

Debian 11

**Comment by jmwmusic** [comment]
Currently with NVDA when I press enter on Appearance (which says it is a button) I then get no further screen reader feedback at all.  I can't seem to arrow or tab in any direction.  This feels the same as what happens with the Palette Add and More and Help/About dialogues.  I was about to report this as a separate issue, but am I just getting stuck in the first spin button when I open it with keyboard.  can someone re-test this without the clicking on the first box step and see if it is possible to navigate this window at all with keyboard?

**Comment by MarcSabatella** [comment]
I think you are indeed stuck in the first spin box ("Leading space"), which is exactly what this issue is about.  As mentioned in the description, you can get out by hitting Enter to accept the value and then Right to move to the next control.  That's the same trick you need for other spin boxes.  I guess that's an inherent downside of not using Tab.

Orca does read the labels on each of these spin boxes, so the info must be there.  The fact that NVDA does not read the labels indicates we probably aren't putting the info everywhere it should be, though.  This seems probably related to https://github.com/musescore/MuseScore/issues/25404

**Comment by jmwmusic** [comment]
No, Pressing enter does not allow me to then press a right arrow (or at least if it is working it is not speaking). In this window I can get no feedback from NVDA at all.  Usually in a spin button it would read the values as you move up or down with arrow keys, that isn't happening either.  It may be navigating, if it is I don't know as there is no speech.  It looks like this might be a bigger issue than just spin buttons. Can this be checked here or should I open a new issue, or add to the ones relating to palette More and Add which seem identical in behaviour?

**Comment by MarcSabatella** [comment]
Yes, as noted above, NVDA doesn’t read anything, but other screen readers do.  The navigation definitely works; it’s the screen reader feedback that is (sometimes) missing.

**Comment by jmwmusic** [comment]
In 4.7 beta the appearance properties area is now reading with NVDA.  Each spin button I get stuck in can be closed with Esc, so it can be used, but this is less than ideal to have to attempt to navigate on each one, potentially moving numbers I didn't want to change before realising I need to press Esc.   Ideally these should only open when a down arrow is pressed on them and then be able to be tabbed away from, or if tab can't be used open and close with space like other combo boxes in MuseScore, not ideal, but at least consistent with the rest of the software.
